assets/src/phoenix/app.ts

```diff
@@ -68,6 +69,8 @@ $(() => {
   ($('.ui.dropdown') as any).dropdown();
   ($('.ui.dropdown.item') as any).dropdown();
 
+  $('[data-action="command-button"]').on('click', commandButtonClicked);
```

Comment:
This is the sketchiest part of this PR that annoys me.

We've got ~3 ways of getting stuff on a page during delivery.

1. Just having phoenix render html
2. Having phoenix instantiate a react component to render
3. phoneix renders html, but it gets augmented jquery style (popovers, this, maybe others?)


I prefer #1 or #2, but this is a component with JS requirements *and* it has children rendered. I don't think #2 is appropriate in those cases?